library/zlib/deflate/deflate_spec.rb

```diff
@@ -1,22 +1,24 @@
-# -*- encoding: us-ascii -*-
```

Comment:
Interesting, so the strings automatically become `ASCII-8BIT` on MRI if they are not valid `US-ASCII`. But this doesn't happen if `utf-8` is used or there is no comment.